javascript-vanilla-v2/main.js

```diff
@@ -0,0 +1,158 @@
+// https://docs.flagsmith.com/clients/javascript#initialisation-options
+// code and comments by @gnumoreno and @MatheusLasserre (github ids)
+const debug = 0
+flagsmith.init({
+  // Dev environment key: iefBdPk5jLjp3WfFjiB4MS
+  // Prod environment key: B7PbVXEWTCGhTD3njUxoL3
+  environmentID: 'iefBdPk5jLjp3WfFjiB4MS', // Required, the environment ID for your environment
+
+  api: 'https://edge.api.flagsmith.com/api/v1/', // Defaults to Flagsmith edge API
+  enableAnalytics: true, // See https://docs.flagsmith.com/flag-analytics/ for more info
+
+  // Using caching options you're limiting your max number of api calls.
```

Comment:
This isn't really true since it's still dependent on the number of users. It's a way of controlling the number of API calls, but it's not a limit.

Reply:
The train of thought was that for a given number of users, you would have a max number of api calls. But this way of thinking really has some caveats. What about i change it to "Using caching options you're reducing the number of api calls for each user."?

Reply:
Yep, that's fine.